miqa/learning/README.md

```diff
@@ -55,7 +55,7 @@ This will convert your pointer files. You should now be able to use the files in
 ## Run inference
 To run inference on a single file, execute:
 ```shell
-python ./miqa/learning/nn_classifier.py -m ./models/miqa01-val0.pth -1 ./PredictHD_small/sub-699312/ses-38845/anat/sub-699312_ses-38845_run-003_BADT1w.nii.gz
+python ./miqa/learning/nn_classifier.py -m ./models/miqaT1-val0.pth -1 ./PredictHD_small/sub-699312/ses-38845/anat/sub-699312_ses-38845_run-003_BADT1w.nii.gz
```

Comment:
All the commits until this one (including this one) have been integrated by de0905c632b377a8bbee32de50fbe8875402fbe7. They have been squashed into a single commit, that is probably what is confusing you. You can (and should) skip this and previous commits on this branch.